tern/analyze/docker/run.py

```diff
@@ -69,6 +69,35 @@ def analyze(image_obj, args, dfile_lock=False, dfobj=None):
         analyze_docker_image(image_obj, args.redo, dfile_lock, dfobj)
 
 
+def execute_oci_image(args):
```

Comment:
We should create a directory called `tern/analyze/oci` and put this is a `run.py` file there.